src/Core/CMakeLists.txt

```diff
@@ -34,7 +35,11 @@ add_library(
 )
 
 add_dependencies( ${core_target} eigen3  openmesh_lib)
-target_compile_definitions(${core_target} PRIVATE "-DRA_CORE_EXPORTS")
+
+# Define the base path for default shader. To be defined according to the install operation.
+message("Setting radium env base dir to ${RADIUM_BINARY_OUTPUT_PATH}")
+
+target_compile_definitions(${core_target} PRIVATE "-DRA_CORE_EXPORTS" RADIUM_BIN_BASEDIR="${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/")
```

Comment:
please be consistent , `${RADIUM_BINARY_OUTPUT_PATH}` or `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/`?